src/Model/User.php

```diff
@@ -304,6 +304,32 @@ private static function updatePasswordHashed($uid, $pasword_hashed)
 		return dba::update('user', $fields, ['uid' => $uid]);
 	}
 
+	/**
+	 * @brief Checks if a nickname is in the list of the forbidden nicknames
+	 *
+	 * Check if a nickname is forbidden from registration on the node by the
+	 * admin. Forbidden nicknames (e.g. role namess) can be configured in the
+	 * admin panel.
+	 *
+	 * @param string $nickname The nickname that should be checked
+	 * @return boolean True is the nickname is blocked on the node
+	 */
+	public static function isNicknameBlocked($nickname)
+	{
+		$forbidden_nicknames = Config::get('system', 'forbidden_nicknames', '');
+		// if the config variable is empty return false
+		if (!x($forbidden_nicknames)) {
+			return false;
+		}
+		// check if the nickname is in the list of blocked nicknames
+		$forbidden = explode(',', $forbidden_nicknames);
+		if (in_array(strtolower($nickname), $forbidden)) {
```

Comment:
You forgot to `trim()` the values, the current default would have " root", " postmaster", etc...

You can do this like `$a = array_map('trim', $a);`

Reply:
I've added the trim.